src/main/kotlin/org/jetbrains/research/ml/coding/assistant/util/psi/Traversals.kt

```diff
@@ -1,3 +1,7 @@
+/*
+ * Copyright (c) 2020.  Anastasiia Birillo
```

Comment:
Should we include my name also since I wrote half of this file?

Reply:
Oh sorry, of course. It is default copyright in my Idea for all my projects, I will change it